src/project.js

```diff
@@ -70,31 +70,51 @@ function projectionReducer(location) {
     newLocation.forestType = forestType;
   }
 
-  newLocation.altitudinalZone = getNextHeigtLevel(location.altitudinalZone);
+  if (location.altitudinalZone && targetaltitudinalZone !== undefined) {
+    newLocation.altitudinalZone = getNextHeigtLevel(location.altitudinalZone);
+  } else {
+    newLocation.altitudinalZone = location.altitudinalZone;
```

Comment:
Is this necessary? I guess this should be coverd by line 44.

Reply:
This solved the problem of tree-app not showing the options for altitudinalZone to choose from. The reason was that if the initial condition is not checked in the library then the options object from returned newLocation will only contain targetHeightLevel as "10" and no other options such as forestEcoregion etc.

Reply:
getNextHeightLevel (changed to getNextAltitudinalZone) moved to line 73 so above if statement is not required anymore.